Expensify/App
# fixed scrolling when focusing on the edit box for old message

### Details


### Fixed Issues

Fixes #3545 

### Tests | QA Steps

  1.  Open any conversation on Ecash Mobile.
  2. Hover over a sent message and click on the edit comment option in the menu
  3. Focus on the edit box, Start editing the message.
  4. When the keyboard opens, you should be able to type in the edit box and chat should not scroll. Like 
  https://user-images.githubusercontent.com/43996225/121611132-92c64480-ca25-11eb-8aaa-8c5204107ff7.mp4


### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/121732118-76063b80-cb0f-11eb-8688-a4605150afcc.mp4
